packages/design-system-tokens/src/themes/cmsgov-component.ts

```diff
@@ -449,11 +449,20 @@ export const components: AnyTokenValues = {
 
   'usa-banner': {
     '__background-color':                         t.color['gray-lightest'],
-    '__color':                                    t.color['black'],
-    '-close__background-color':                   t.color['gray-light'],
-    '-action__color':                             t.color['primary-darker'],
-    '-lock-icon__color':                          t.color['success-light'],
-    '-heading__font-size':                        '0.75rem',
+    '__color':                                    t.color['base'],
+    '__max-width':                                t.global['site-max-width'],
+    '-gutter-width':                              t.global['grid-gutter-width'],
+    '-heading__font-family':                      t.font['sans'],
+    '-heading__line-height':                      '1.1',
+    '-heading__font-size':                        t.font['size-sm'],
```

Comment:
Our original implementation used `.75rem` (12px) as the font-size for the banner heading text. USWDS is slightly larger at `.8rem` (12.8px). The smallest font size in our DS was `.875rem` (14px) and I used that for consistency - can easily change though.

Reply:
This change seems fine to me, but we should probably run it by @brentschneider when he's back. Not sure if any stakeholders will want to know about the height difference ahead of time.